cypress/support/constants/constants.js

```diff
@@ -52,6 +52,7 @@ module.exports = {
   COMBINEDFIREBOLTMOCKS: 'combinedFireboltMocks',
   COMMUNICATION_MODE: 'communicationMode',
   CONFIG: 'CONFIG',
+  CONFIG_MODULE_SETRESPONSE_PATH: './node_modules/configModule/testData/setResponseData.json',
```

Comment:
Why is this `node_modules` and not `external`?

Reply:
After consulting Kurt about retrieving the JSON files from external sources or nodeModules, he confirmed that it's okay to read them from nodeModules.

Reply:
Tested the changes from `fixtures/external`, it's working so keeping the logic to read from external.